### DISCUSSION TITLE
Questa 2021.3 supports VHPI

### DISCUSSION THREAD

**Discussion by raysalemi** [cocotb/cocotb#2694] [Proposals]
Hi All,

We (Siemens) believe that Questa 2021.3 should support cocotb using VHPI. 

Would you be able to add this combination to the test suite?

Ray

**Comment by ktbarrett** [comment]
@raysalemi That's great news! @imphil Could the private CI be updated to the new version of Questa? Then we can add a new test to the regression for the VHPI interface.

**Reply by imphil** [reply]
We have been running 2021.3 for a while for extended regressions, and now also run 2021.4. There are still some issues with Questa VHPI and/or cocotb working together, https://github.com/cocotb/cocotb/issues/2803 tracks them.
